website/frontend/templates/docs/scripting.html

```diff
@@ -174,6 +174,8 @@ <h3 id="unset_name">$unset(name)</h3>
           <li>Unsets the variable
             <code>name</code>.
           </li>
+          <li>Unsets a tag (i.e. <code>$unset(performer:*)</code> would unset all performer tags)
```

Comment:
We could maybe improve the description a bit so that wildcard matching is evident? And it only works for certain tags - 
https://github.com/metabrainz/picard/commit/6cc8ebce725b1281bcd5dde688a223ef7098d3fa#diff-49991922d5e1bb94a02cf28bf4619a1eR426
Maybe you can throw a not in there for the 3 tags it works with.